packages/editor/src/client/locales/rsuiteEn.ts

```diff
@@ -0,0 +1,65 @@
+// const Calendar = {
```

Comment:
Why did you comment this out here?

Reply:
This file will be deleted, I don't know why it hasn't been when I updated the branch.